client/src/components/Icons/SlackIcon.css

```diff
@@ -0,0 +1,12 @@
+
+.icons-container {
```

Comment:
Doing a display flex with one thing in the container doesnt really do much. but the size of the icon is good